express-post-json/index.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const app = express();
+let nextId = 1;
+const grades = {};
+const gradesArray = [];
+
+app.get('/api/grades', (req, res) => {
+  res.json(gradesArray);
+});
```

Comment:
It's best to build this array inside of the `app.get()` callback on-demand instead of trying to maintain two data models (an object and an array)